executable/optionrom.ksy

```diff
@@ -0,0 +1,474 @@
+meta:
+  id: optrom
```

Comment:
I agree with @GreyCat, `firmware` sounds more reasonable to me than `executable`.